src/main/resources/application.properties

```diff
@@ -180,4 +180,32 @@ quality-Audit-PageSize=5
 ## max no of failed login attempt
 failedLoginAttempt=5
 
+###CTI data sync schedular
+cz-duration=40
+## prescription template
+TMprescriptionTemplate=TMPrescription
+
+##duration for everwell calender
+everwellCalendarDuration=15
+ 
+##lonic variables
+lonicPageSize=10
+ 
+#snomedcti
+snomedCTPageSize=10
+
+#call retry count
+callRetryConfiguration=3
+##---------------------------------------------------------------------------------------------------------------
+#avni registration Duration
+avniRegistrationLimit=7
+
+#--------------------------NHM Agent Real Time Data----------------------------
+nhm.agent.real.time.data.url= http://175.101.1.83/apps/utility/alive_api.php
```

Comment:
Should we be hardcoding this IP?

@devikasuresh20, @ravishanigarapu what's this about?

Reply:
Removed the hardcoded ip.